### DISCUSSION TITLE
Fast turn on and off

### DISCUSSION THREAD

**Discussion by danieledalsanto** [maca88/SmartBikeLights#4] [Ideas]
Hi, I'm an owner of an Edge 530. I would like to suggest a feature I would find useful. I would like to have the possibility to turn on and off all the lights using a shortcut (like holding the menu button) instead of having to reach the data field section and individually turn on and off the lights (I'm looking for something that is faster than operating directly on the lights). It is not a big deal at all if this is not feasible. Thanks for the app and the attention.
Daniele.

**Comment by maca88** [comment]
Hi,

unfortunately data field applications do not have the ability to detect a button press, so the only way to fast turning the lights off/on is to use the provided filters. You can check #3 as there we had a similar discussion.
